aws/v11.4.2/release.yaml

```
@@ -0,0 +1,60 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v11.4.2
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.2.3
+  - componentVersion: 0.9.0
+    name: cert-manager
+    version: 1.0.8
+  - name: chart-operator
+    version: 0.12.4
+  - componentVersion: 1.16.5
+    name: cluster-autoscaler
+    version: 1.16.0
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.1.10
```

```suggestion
    version: 1.2.0
```
Please consider updating coredns app - includes readiness probe + load shedding by making liveness probe more resilient than readiness probe; see changelog https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v120-2020-07-13

//cc @stone-z

---

Did you test the app? If it's tested we can include it in the release.

---

coredns 1.2.0 was tested with and included in AWS 9.3.4 platform release that was released yesterday.

So, it should work with 11.4.2, but you never know.

@giantswarm/team-ludacris as coredns-app owners, AFAIK no special/exotic coredns specific test is needed, as long as coredns Pods come up and everything else works in the release.

I typically test cluster creation and upgrade from previous commonly used or active platform release.

---

Okay, got it. :) Firecracker is also going to test the release so there shouldn't be a problem.